src/covid_model_seiir_pipeline/forecasting/model/ode_forecast.py

```diff
@@ -197,6 +199,7 @@ def __init__(self,
         self.gamma1 = constants.gamma1
         self.gamma2 = constants.gamma2
         self.N = constants.N
+        self.system_params = constants.system_params
```

Comment:
Suggest using a `.copy()` of this since it's a mutable dict

Reply:
Good idea